toolkit/scripts/prerequisites.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
```

Comment:
This script seems overly complicated.

Would it make sense to have `install-prerequisites-ubuntu.sh` and `install-prerequisites-mariner.sh` scripts and then just have the Markdown reference those 2 scripts? That way, we still have a single source of truth but with significantly fewer lines of code.

Reply:
Per offline discussion with a few people, we are ok to move entirely to this script for installing deps. Still going to leave the customization stuff in the .md file. Script will only handle packages.